specs/marketplace.md

```diff
@@ -185,178 +174,159 @@ struct Content {
 
 ```
 
-Notes about some parameters:
+The meaning of the Request parameters is given below:
+
+> Note that not all parameters are defined. I think that in a formal spec we need to introduce all of them, even shortly.
 
 `cid` 
 
 An identifier used to locate the Manifest representing the dataset.
-- MUST be a [CIDv1](https://github.com/multiformats/cid#cidv1) with sha-256 based [multihash](https://github.com/multiformats/multihash).
-- Data it represents SHOULD be discoverable in the network, otherwise the Request will get canceled. 
+- MUST be a [CIDv1](https://github.com/multiformats/cid#cidv1), SHA-256 [multihash](https://github.com/multiformats/multihash).
+- The data it represents SHOULD be discoverable in the network; otherwise, the Request MUST be canceled.
 
 `reward`
 
-- It MUST be an amount of tokens offered per slot per second.
-- The Ethereum address that submits the `requestStorage()` transaction MUST have [approval](https://docs.openzeppelin.com/contracts/2.x/api/token/erc20#IERC20-approve-address-uint256-) for transfer of at least the same amount on the ERC20 based token, that the network uses. 
+- MUST be an amount of Codex Tokens offered per slot per second.
+- The Ethereum address that submits the `requestStorage()` transaction MUST have [approval](https://docs.openzeppelin.com/contracts/2.x/api/token/erc20#IERC20-approve-address-uint256-) for the transfer of at least an equivalent amount in Codex Tokens.
+
+> The second item could have a better explanation. E.g., it is not clear, who is the `approver`.
 
 `collateral`
 
-- Amount of tokens that the storage providers submits when they fill slots.
-- Collateral is then slashed or forfeited if the storage providers fail to provide the service requested by the Request (more information below).
+- The amount of tokens that Storage Providers submit when they fill slots.
+- Collateral is then slashed or forfeited if Storage Providers fail to provide the service requested by the Request (more information below).
 
 `proofProbability`
 
-Determines the average frequency that a proof is required in a period: $\frac{1}{proofProbability}$
+Determines the average frequency that a proof is required within a period: $\frac{1}{proofProbability}$
+
+> Does it have a unit?
 
-- Storage providers are required to provide proofs of storage to the marketplace smart contract when they are challenged by the smart contract.
-- The frequency is stochastic in order to prevent hosts from only coming online when proofs are required, but it is affected by this parameter.
+- Storage Providers are required to provide proofs of storage to the Marketplace Smart Contract when prompted by the Smart Contract.
```

Comment:
In this context, a "challenge" is more technically correct than "prompt", as "prompt" tends to imply an interaction. The "challenge" referred to here is the proof requirement.